### DISCUSSION TITLE
Hide some upgrades

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-pkgs#40114] [Q&A]
is there any way to to hide/ignore upgrades from some packages?

here's the problem.

```
 $ winget upgrade
Name                                            Id                           Version        Available      Source
-----------------------------------------------------------------------------------------------------------------
Microsoft Visual C++ 2008 Redistributable - x86 Microsoft.VC++2008Redist-x86 9.0.30729.4148 9.0.30729.6161 winget
Microsoft Visual C++ 2008 Redistributable - x64 Microsoft.VC++2008Redist-x64 9.0.30729.4148 9.0.30729.6161 winget
Microsoft .NET SDK                              Microsoft.dotnet             5.4.321.52708  6.1.121.56904  winget
3 upgrades available.
```

**Comment by OfficialEsco** [comment]
Related issues:
* https://github.com/microsoft/winget-pkgs/issues/17985
* https://github.com/microsoft/winget-pkgs/issues/38045
* https://github.com/microsoft/winget-pkgs/issues/27935

I believe you are looking for this feature https://github.com/microsoft/winget-cli/issues/476 which is in the [client version 1.3 milestone](https://github.com/microsoft/winget-cli/milestone/37)
